### PR TITLE
Use official redis charts

### PR DESCRIPTION
**Pull request by leafty**

### CODE REVIEW

File: charts/renku/values.yaml [hunk 1]
```diff
@@ -450,32 +467,17 @@ gitlab:
...
-  ## Horizontal scaling of pods
-  # replicaCount: 1
-  # antiAffinity: soft
+  usePassword: false
 
-  ## Resource requests/limits for Redis
-  # resources: {}
+  master:
+    persistence:
+      enabled: false
```
Comment (leafty) [inline charts/renku/values.yaml]: TODO: Network policy.

**Comment by rokroskar** [comment]
@leafty can you please resolve conflicts here so it can be merged?

**Comment by leafty** [comment]
It's a good idea to have it

File: charts/renku/requirements.yaml [hunk 2]
```diff
@@ -1,29 +1,30 @@
 dependencies:
 - name: renku-ui
   alias: ui
-  version: "0.1.0-c776f4e"
+  version: "0.1.0-0bc7625"
```
Comment (leafty) [inline charts/renku/requirements.yaml]: These are upgrades, not downgrades?

File: charts/renku/values.yaml [hunk 3]
```diff
@@ -493,32 +493,20 @@ gitlab:
...
-  # service:
-  #   port: 6379
-  #   type: ClusterIP
+  master:
+    persistence:
+      enabled: false
+      
+  networkPolicy:
+    enabled: true
+    allowExternal: false
```
Comment (leafty) [inline charts/renku/values.yaml]: Works as intended (only GitLab container can contact redis)

File: charts/renku/requirements.yaml (hunk 2 above)
Comment (rokroskar) [reply charts/renku/requirements.yaml]: yes -- were they supposed to be downgrades?

Comment (leafty) [reply charts/renku/requirements.yaml]: nah, i just didn't check the git logs to verify that

**Pull request merged**